### ISSUE TITLE
"LookupError: unknown encoding: 65001" while activating envs

### ISSUE THREAD

**Issue by WSQi** [conda/conda#8327]
## Current Behavior
After Checking Windows 10 *Language for non-Unicode programs* > **Beta: Use Unicode UTF-8 for worldwide language support**

`Anaconda Prompt` will throw error while activating environment:
```
Fatal Python error: init_sys_streams: can't initialize sys standard streams
LookupError: unknown encoding: 65001
```

PS: Please noted that it was not a duplicate with ticket #5448, I'm using `Anaconda Prompt` while triggering the error.

### Steps to Reproduce

1. Open `Control Panel\Clock and Region\Region` > Administrative Tab > Change system locale
1. Check Windows 10 **Beta: Use Unicode UTF-8 for worldwide language support**
![image](https://user-images.githubusercontent.com/5621964/53282316-0d91c580-3771-11e9-8a9c-9c5615ed6119.png)
1. Restart Computer
1. Open `Anaconda Prompt` from start menu
1. type `conda activate any_envs`

### Workaround
Execute `set PYTHONIOENCODING=UTF-8` before activate any environments

![image](https://user-images.githubusercontent.com/5621964/53282298-da4f3680-3770-11e9-85b6-7293124c933a.png)

## Expected Behavior

Environment should be successfully activated

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
(base) C:\Users\conta>conda info

     active environment : base
    active env location : D:\Apps\miniconda3
            shell level : 1
       user config file : C:\Users\conta\.condarc
 populated config files :
          conda version : 4.6.4
    conda-build version : not installed
         python version : 3.7.1.final.0
       base environment : D:\Apps\miniconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : D:\Apps\miniconda3\pkgs
                          C:\Users\conta\.conda\pkgs
                          C:\Users\conta\AppData\Local\conda\conda\pkgs
       envs directories : D:\Apps\miniconda3\envs
                          C:\Users\conta\.conda\envs
                          C:\Users\conta\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.6.4 requests/2.21.0 CPython/3.7.1 Windows/10 Windows/10.0.17763
          administrator : False
             netrc file : None
           offline mode : False


(base) C:\Users\conta>
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
(base) C:\Users\conta>conda config --show-sources

(base) C:\Users\conta>
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
(base) C:\Users\conta>conda list --show-channel-urls
# packages in environment at D:\Apps\miniconda3:
#
# Name                    Version                   Build  Channel
altgraph                  0.16.1                   pypi_0    pypi
asn1crypto                0.24.0                   py37_0    defaults
ca-certificates           2019.1.23                     0    defaults
certifi                   2018.11.29               py37_0    defaults
cffi                      1.11.5           py37h74b6da3_1    defaults
chardet                   3.0.4                    py37_1    defaults
conda                     4.6.4                    py37_0    defaults
conda-env                 2.6.0                         1    defaults
console_shortcut          0.1.1                         3    defaults
cryptography              2.4.2            py37h7a1dbc1_0    defaults
entrypoints               0.3                      pypi_0    pypi
flake8                    3.7.4                    pypi_0    pypi
future                    0.17.1                   pypi_0    pypi
idna                      2.8                      py37_0    defaults
macholib                  1.11                     pypi_0    pypi
mccabe                    0.6.1                    pypi_0    pypi
menuinst                  1.4.14           py37hfa6e2cd_0    defaults
mypy                      0.660                    pypi_0    pypi
mypy-extensions           0.4.1                    pypi_0    pypi
natsort                   5.5.0                    pypi_0    pypi
openssl                   1.1.1a               he774522_0    defaults
pefile                    2018.8.8                 pypi_0    pypi
pip                       18.1                     py37_0    defaults
pycodestyle               2.5.0                    pypi_0    pypi
pycosat                   0.6.3            py37hfa6e2cd_0    defaults
pycparser                 2.19                     py37_0    defaults
pyflakes                  2.1.0                    pypi_0    pypi
pyinstaller               3.4                      pypi_0    pypi
pyopenssl                 18.0.0                   py37_0    defaults
pysocks                   1.6.8                    py37_0    defaults
python                    3.7.1                h8c8aaf0_6    defaults
pywin32                   223              py37hfa6e2cd_1    defaults
pywin32-ctypes            0.2.0                    pypi_0    pypi
requests                  2.21.0                   py37_0    defaults
ruamel_yaml               0.15.46          py37hfa6e2cd_0    defaults
setuptools                40.6.3                   py37_0    defaults
six                       1.12.0                   py37_0    defaults
sqlite                    3.26.0               he774522_0    defaults
typed-ast                 1.2.0                    pypi_0    pypi
urllib3                   1.24.1                   py37_0    defaults
vc                        14.1                 h0510ff6_4    defaults
vs2015_runtime            14.15.26706          h3a45250_0    defaults
wheel                     0.32.3                   py37_0    defaults
win_inet_pton             1.0.1                    py37_1    defaults
wincertstore              0.2                      py37_0    defaults
yaml                      0.1.7                hc54c509_2    defaults

(base) C:\Users\conta>
```
</p></details>

**Comment by Robert-Lu** [comment]
I have the same issue when using the command `activate` if the `Beta: Use Unicode UTF-8 for worldwide language support` option is checked in the Windows Settings. Now I have to turned off this option and everything works fine.

When the option is on, my console gives 65001 for command `chcp`, and now gives 437.  Also, this issue should be invariant for terminal simulator, I have the same issue for CMDer, original cmd.exe and Power Shell.

**Comment by WSQi** [comment]
Seems to be fixed by merge request #8342.
